priv/templates/boot.eex

```diff
@@ -208,6 +208,15 @@ _replace_os_vars() {
         }1' < "$1" > "$2"
 }
 
+# Private. Run hooks from directory.
+_run_hooks_from_dir() {
+   if [ -d "$1" ]; then
+          for s in $(find "$1" -name "*.sh" -print | sort); do
```

Comment:
Excellent! I gave a similar `find` line a try yesteday, and I couldn't make it work on my OSX laptop. Looking now, I had an error in there (opened the name pattern with `^` as if it was a grep pattern) which made me think `find` on OSX couldn't do it.